lib/pages/portal/service/website_provider.dart

```diff
@@ -96,13 +100,32 @@ class WebsiteProvider with ChangeNotifier {
     print(e.message);
     if (context != null) {
       if (e.message.contains('PERMISSION_DENIED')) {
-        AppToast.show(S.of(context).errorPermissionDenied);
+        AppToast.show(S
+            .of(context)
+            .errorPermissionDenied);
       } else {
-        AppToast.show(S.of(context).errorSomethingWentWrong);
+        AppToast.show(S
+            .of(context)
+            .errorSomethingWentWrong);
       }
     }
   }
 
+  void getVisits(List<Website> websites) {
+    var websitesId = PrefService.sharedPreferences.getStringList("websitesId");
```

Comment:
Avoid `var`s unless it's super obvious what type that variable is (e.g. `var website = Website()` is acceptable).